src/validator-map.js

```diff
@@ -77,8 +77,12 @@ function isFloat(value) {
 VALIDATOR_MAP[DATA_TYPES.FLOAT] = isFloat;
 
 // 1, 2.2, 3.456789e+0
+const zeroPaddedNumCheck = Utils.buildRegexCheck('isZeroPaddedNumber');
```

Comment:
Nit: Comment above should now move down?
Nit 2: It looks like this just avoid number parsing - should we explicitly output string in this case, rather than having it fall through the number check?

Reply:
yea, but that's not how `computeColMeta` currently designed. with the current implementation 
`type = allValidators.find(buildValidatorFinder(data, columnName));` The only way to have it return `string` it to fell through the number check, because string is at the bottom of the validator queue